### PR TITLE
Broken Link Updated. 

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fixed the broken link with modification of lines #33926

### CODE REVIEW

**Comment by NitishKumar06** [comment]
Hey @davidopp @kevin-wangzefeng @Shubham82  . Please go through this PR. The broken link has been updated now. Feel free to suggest some changes again if necessary.

File: content/en/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 1]
```diff
@@ -302,9 +302,8 @@ the Pod onto a node that is in the same zone as one or more Pods with the label
 `topology.kubernetes.io/zone=R` label if there are other nodes in the
 same zone currently running Pods with the `Security=S2` Pod label.
 
-See the
-[design doc](https://git.k8s.io/community/contributors/design-proposals/scheduling/podaffinity.md)
-for many more examples of Pod affinity and anti-affinity.
+To get yourself more familiar with the examples of Pod affinity and anti-affinity
```
Comment (tengqm) [inline content/en/docs/concepts/scheduling-eviction/assign-pod-node.md:305]:
```suggestion
To get yourself more familiar with the examples of Pod affinity and anti-affinity,
```

File: content/en/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 2]
```diff
@@ -302,9 +302,8 @@ the Pod onto a node that is in the same zone as one or more Pods with the label
...
+Refer to this [link](https://github.com/kubernetes/design-proposals-archive/blob/main/scheduling/podaffinity.md)
```
Comment (tengqm) [inline content/en/docs/concepts/scheduling-eviction/assign-pod-node.md:306]:
```suggestion
refer to the [design proposal](https://github.com/kubernetes/design-proposals-archive/blob/main/scheduling/podaffinity.md).
```

**Comment by NitishKumar06** [comment]
@tengqm Is it fine now?

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Comment by NitishKumar06** [comment]
You're welcome @tengqm . I got my first PR merged.

**Pull request merged**